cli/src/main/java/org/jboss/as/cli/impl/CommandContextImpl.java

```diff
@@ -523,18 +565,26 @@ private void restoreStdIO() {
     }
 
     private void initCommands() throws CommandLineException, CommandLineParserException {
+        initCommands(false);
+    }
+
+    private void initCommands(boolean bootInvoker) throws CommandLineException, CommandLineParserException {
         // aesh commands
         cmdRegistry.addCommand(new VersionCommand());
         cmdRegistry.addCommand(new HelpCommand(cmdRegistry));
-        cmdRegistry.addCommand(new ConnectCommand());
+        if (!bootInvoker) {
```

Comment:
stylistic suggestion: maybe you should add things that change with `bootInvoker` in a single `if (!bootInvoker)`?

Reply:
@jmesnil I would prefer to keep the current order when registering commands.